solarkraft/src/fetcher/callDecoder.ts

```diff
@@ -50,6 +50,7 @@ export async function extractContractCall(
     // `created_at` and `env.ledger().timestamp()` have seconds-precision. We divide by 1000 here since `Date` internally
     // represents the timestamp as milliseconds.
     const timestamp = new Date(op.created_at).getTime() / 1000
+    const transaction_successful = op.transaction_successful
```

Comment:
This is a long name. How about `tx_ok` (or `txOk` since we have `txHash`)?

Reply:
I've shortened it to `txSuccess` (I don't think `txOk` conveys enough meaning)